README.md

```diff
@@ -34,7 +34,7 @@ We would love your feedback on how to improve the contribution experience, pleas
 
 ## License
 
-Electricity Maps is licensed under GNU-AGPLv3. Find our license [here](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/LICENSE.md)
+This repository is licensed under GNU-AGPLv3 since v1.5.0 released on 31st January 2023. Find our license [here](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/LICENSE.md). Prior contributions were licensed under [MIT license](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/LICENSE_MIT.txt)
```

Comment:
We should specify the exact commit instead of the release version here.

Reply:
I have a hard time finding which commit we should specify here, do you have a suggestion?

Reply:
The exact commit is cb9664f43f0597bedf13e832047c3fc10e67ba4e but I would add it as `[cb9664f](https://github.com/electricitymaps/electricitymaps-contrib/commit/cb9664f43f0597bedf13e832047c3fc10e67ba4e)` in consistency with GitHub commit referencing.